config/values.yaml

```diff
@@ -108,6 +108,7 @@ controlPlane:
         requests:
           cpu: 40m
           memory: 64Mi
+      version: ""
```

Comment:
I think we should move that up, so people see it sooner, just move it below enabled

Reply:
Sounds good, I thought they're sorted alphabetically:) fixed!